pom.xml

```
@@ -3,7 +3,7 @@
 
   <groupId>org.spdx</groupId>
   <artifactId>spdx-maven-plugin</artifactId>
-  <version>0.7.5-SNAPSHOT</version>
+  <version>0.7.6-SNAPSHOT</version>
```

```suggestion
  <version>0.7.5-SNAPSHOT</version>
```

I would prefer not to bump the release version prior to release